### DISCUSSION TITLE
Sortable Context using dnd sortables

### DISCUSSION THREAD

**Discussion by khanzzirfan** [samuelarbibe/dnd-timeline#57] [Q&A]
Hello ,
I would like to implement the sortable context and available strategy. Is there way to implement it in the dnd-timeline?
I want to implement the horizontal and vertical sortable strategy for swapping the timeline elements.

For example, 
https://docs.dndkit.com/presets/sortable/sortable-context#strategy

a code sandbox
https://codesandbox.io/p/sandbox/dnd-kit-multi-containers-forked-gwx6gr?file=%2Fsrc%2Fcontainer2.js%3A25%2C6&workspaceId=0628b686-91f5-415f-b24d-97384f3a74f6

**Comment by samuelarbibe** [comment]
Hey!  
Unfortunately, I cannot think of a way to implement a horizontal dnd sorting strategy.   
It sounds like you need more of a grid rather than a timeline

Good luck 🤞
